### DISCUSSION TITLE
Addons panel is not shown in Safari frequently

### DISCUSSION THREAD

**Discussion by jasikpark** [storybookjs/storybook#25798] [Help] [answered]
### Summary

I am using Storybook + Vite at work and frequently cannot show the addons panel when using the app

### Additional information

https://github.com/storybookjs/storybook/assets/10626596/5fea09b9-0356-4476-a147-89eb391b79bc

### Create a reproduction

n/a / replicate by opening https://stackblitz.com/github/storybookjs/sandboxes/tree/next/react-vite/default-ts/after-storybook?'s iframe in a separate tab

**Comment by valentinpalkovic** [comment, marked as answer]
This is a known bug in Storybook 7 and will be fixed in Storybook 8. You can try out now the latest Storybook 8 alpha if you want by running `npx storybook@next upgrade`

**Comment by valentinpalkovic** [comment]
To fix the issue in Storybook 7, it's necessary to delete your local storage.
